code/scripting/api/objs/shipclass.cpp

```diff
@@ -1145,6 +1145,81 @@ ADE_FUNC(renderTechModel2, l_Shipclass, "number X1, number Y1, number X2, number
 	return ade_set_args(L, "b", true);
 }
 
+ADE_FUNC(renderSelectModel,
+	l_Shipclass,
+	"shipclass, boolean restart, number x, number y, [number width = 629, number height = 355, number = currentEffectSetting]",
```

Comment:
usually, the member that the function is called on (i.e. the shipclass in this case) is not part of the list of explicit arguments (since you write ``shipclass:renderSelectModel(restart ...)`` in lua). So just remove that first shipclass in this documentation string and you're good.

Reply:
Right. I think I originally had these under UI but it felt better here. Anyway. Fixed!